src/librustc_codegen_llvm/consts.rs

```diff
@@ -64,13 +64,16 @@ fn set_global_alignment(cx: &CodegenCx,
 pub fn addr_of_mut(cx: &CodegenCx,
                    cv: ValueRef,
                    align: Align,
-                   kind: &str)
+                   kind: &str,
+                   addr_space: Option<AddrSpaceIdx>)
```

Comment:
There shouldn't be that many calls, making this non-optional (and using `AddrSpaceKind`) would be better IMO.